client/galaxy/scripts/apps/login.js

```diff
@@ -0,0 +1,39 @@
+
+var jQuery = require( 'jquery' ),
+    $ = jQuery,
+    PANEL = require( 'layout/panel' ),
+    _l = require( 'utils/localization' ),
+    PAGE = require( 'layout/page' );
+
+window.app = function app( options, bootstrapped ){
```

Comment:
On another note, currently the Login and the Analysis view are loaded without refreshing the whole page ( Workflow, Pages and so on were to follow soon ) see e.g. https://github.com/galaxyproject/galaxy/blob/dev/client/galaxy/scripts/mvc/app/app-view.js#L57. Without this feature preparing on-screen tours might be tricky. Will that be still possible after merging this?